spec/system/candidate_interface/automatically_carry_over_unsubmitted_application_to_new_cycle_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'rails_helper'
+
+RSpec.feature 'Automatically carry over unsubmitted applications' do
+  include CandidateHelper
+
+  around do |example|
+    Timecop.freeze(Date.new(2020, 8, 1)) do
```

Comment:
```suggestion
    Timecop.freeze(Date.new(RecruitmentCycle.current_year, 8, 1)) do
```

Reply:
I'm going to keep this hard-coded date in the test because it needs to be relative to the dates defined in `EndOfCycleTimetable`.

Reply:
I have removed the hard-coded date in the implementation and changed `EndOfCycleTimetable` to use `RecruitmentCycle`.